Base/res/color-palettes/default.palette

```diff
@@ -0,0 +1,28 @@
+#000000
+#808080
+#800000
+#808000
+#008000
+#008080
+#000080
+#800080
+#808040
+#004040
+#0080ff
+#004080
+#8000ff
+#804000
+#ffffff
+#c0c0c0
+#ff0000
+#ffff00
+#00ff00
+#00ffff
+#0000ff
+#ff00ff
+#ffff80
+#00ff80
+#80ffff
+#8080ff
+#ff0080
+#ff8040
```

Comment:
It would be nice to have some newlines at the end of these files, since they are text files (and that's how they're saved in most editors including our own TextEditor)

Reply:
Done :+1: